docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: '3'
+
+volumes:
+  database2: {}
+
+services:
+  # Named "database2" because "database" conflicts with directory inside project
+  database2:
+    image: mysql:latest
+    environment:
+      MYSQL_DATABASE: budget
+      MYSQL_USER: budget
+      MYSQL_PASSWORD: test
+      MYSQL_RANDOM_ROOT_PASSWORD: 'true'
+    command: --default-authentication-plugin=mysql_native_password
+    volumes:
+      - database2:/var/lib/mysql
+
+  php:
+    build: ./
+    environment:
+      DB_HOST: database2
+      DB_DATABASE: budget
+      DB_USERNAME: budget
+      DB_PASSWORD: test
+    working_dir: /usr/share/nginx/budget
```

Comment:
I would put that in the Dockerfile.

```
WORKDIR /usr/share/nginx/budget
```

Reply:
What's the advantage of doing that?

Reply:
It also works if you are not using the Dockerfile and it puts it at the right place.

Reply:
Fair enough, will update this.